.changeset/config.json

```diff
@@ -0,0 +1,15 @@
+{
+	"$schema": "https://unpkg.com/@changesets/config/schema.json",
+	"changelog": [
+		"@changesets/changelog-github",
```

Comment:
[ask] Does `@changesets/changelog-github` match our existing CHANGELOG format? Isn't a custom formatter necessary?

Reply:
The default format is different.

From https://github.com/stylelint/stylelint/issues/6253#issuecomment-1216039310

> We can update CHANGELOG.md automatically via the two things above, right? Additionally, do we need to write a custom formatter to keep the existing CHANGELOG.md format?

If we want to keep the existing CHANGELOG.md format, then we need to write a custom formatter.

Reply:
I see. I think keeping the format is better because it will be easier if we revert the changes.

Is writing a custom formatter hard (needs many lines of code)?

Reply:
> Is writing a custom formatter hard (needs many lines of code)?

In my imagine, it should not be tough, I'll add a custom one in this PR later.